### PR TITLE
[kodi] Update README.md

### PR DESCRIPTION
**Pull request by greengameplayer**
Updated README.md

Minor grammatical fixes and punctuation additions.

### CODE REVIEW

**Comment by J-N-K** [comment]
Thanks. Can you please folow the instructions https://github.com/openhab/openhab2-addons/pull/6178/checks?check_run_id=249093810 to sign-off your work? Thanks.

**Comment by greengameplayer** [comment]
> Thanks. Can you please folow the instructions https://github.com/openhab/openhab2-addons/pull/6178/checks?check_run_id=249093810 to sign-off your work? Thanks.

I did, for some reason it expects a github noreply sign-off email. If you look at it it says: `Commit sha: 3a8a93b, Author: greengameplayer, Committer: GitHub; Expected "greengameplayer 35043196+greengameplayer@users.noreply.github.com", but got "James Rauworth bobfacegreengameplayer@gmail.com".`

File: bundles/org.openhab.binding.kodi/README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 It allows users to play and view most videos, music, podcasts, and other digital media files from local and network storage media and the internet.
 
-The Kodi Binding integrated Kodi media center support with openHAB, allowing both controlling the player as well as retrieving player status data like the currently played movie title.
+The Kodi Binding integrates Kodi media center support with openHAB, allowing both controlling the player as well as retrieving player status data like the currently played movie title.
 
 The Kodi binding is the successor to the openHAB 1.x xbmc binding.
 
 ## Preparation
 
-In order to allow control of Kodi by this binding, you need to enable the Kodi application remote control feature.
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.kodi/README.md]: One minor question:

What was wrong with this part?

The new sentence is definitely wrong. The correct spelling should be "controlled by" or "controlled via".

Comment (greengameplayer) [reply bundles/org.openhab.binding.kodi/README.md]: The sentence seemed a little awkward to me, thanks for catching the mistake.

Comment (greengameplayer) [reply bundles/org.openhab.binding.kodi/README.md]: > "controlled via".
>
Seems perfect here

**Pull request merged**